lib/modules/manager/velaci/extract.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('modules/manager/velaci/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('should handle invalid file', () => {
+      const res = extractPackageFile('\0');
+      expect(res).toBeNull();
+    });
+
+    it('should handle invalid YAML', () => {
+      const res = extractPackageFile(Fixtures.get('invalid.yml'));
```

Comment:
```suggestion
      const res = extractPackageFile('foo:bar:invalid\n' );
```
We don't need a file for this simple case 😉